INSTALL.md

```diff
@@ -14,27 +14,31 @@ Install the following packages with your package manager:
 python3
 python3-gi
 python3-opengl
+python3-yaml
 gir1.2-gtk-3.0
 ```
 
 On a Debian or Ubuntu system, you would just type the following:
 ```bash
-sudo apt install python3-gi python3-opengl gir1.2-gtk-3.0
+sudo apt install python3-gi python3-opengl python3-yaml gir1.2-gtk-3.0
 ```
 Please note that you need to enable the `universe` repository in Ubuntu.
 
 ### Run with Docker
 
 If you have difficulty with the installation, you can run the application from Docker.
 
+The `docker run` command will mount your personal Documents folder to `/root/Documents` so that you
+can access your files.
+
 ```bash
-sudo docker pull pycam/pycam  # downloads the latest image
+sudo docker build -t pycam/pycam .
 sudo docker run -it \
     -v ~/Documents:/root/Documents \
     -v ~/.Xauthority:/root/.Xauthority \
     -e DISPLAY \
     --net=host \
-    pycam/run_gui.py
+    pycam/pycam
```

Comment:
Shouldn't this be `pycam/run_gui.py`?

Reply:
Great question. At this point, we're actually specifying the docker image to use. We named it `pycam/pycam` when we built it in the previous step.

It looks like the path to the executable script but we're actually providing that in the last step of the Dockerfile. If you did want to run a script outside of the default one that we specified, you can do so as such: `pycam/pycam pycam/run_gui.py`. I do that all the time when testing the image (`pycam/pycam /bin/bash`).

Reply:
OK - thanks for your explanation. Thus I recently messed up this line, when I changed the name of the start script. I am sorry!

Reply:
Not a problem 🙂